it/v2/endpoints/RetrieveUkPropertyPeriodSummaryControllerISpec.scala

```diff
@@ -256,7 +286,7 @@ class RetrieveUkPropertyPeriodSummaryControllerISpec extends V2IntegrationBaseSp
 
     "return ifs service error" when {
       def serviceErrorTest(ifsStatus: Int, ifsCode: String, expectedStatus: Int, expectedBody: MtdError): Unit = {
-        s"ifs returns an $ifsCode error and status $ifsStatus" in new Test {
+        s"ifs returns an $ifsCode error and status $ifsStatus" in new NonTysTest {
```

Comment:
Can we update `ifsCode` and `ifsStatus` to `downstreamCode` and `downstreamStatus`